### PR TITLE
Edits to text in Bulk API doc

### PR DESCRIPTION
**Pull request by dmeiss**
Minor edits to text for punctuation and readability.

### CODE REVIEW

**Comment by danielmitterdorfer** [comment]
@elasticmachine test this please

File: docs/reference/docs/bulk.asciidoc [hunk 1]
```diff
@@ -246,20 +246,20 @@ NOTE: Only the shards that receive the bulk request will be affected by
...
-refresh. The other two shards of that make up the index do not
+refresh. The other two shards that make up the index do not
 participate in the `_bulk` request at all.
 
 [float]
 [[bulk-update]]
 === Update
 
-When using `update` action `retry_on_conflict` can be used as field in
+When using `update`, the action `retry_on_conflict` can be used as a field in
```
Comment (danielmitterdorfer) [inline docs/reference/docs/bulk.asciidoc]: In this case we're altering the meaning: `retry_on_conflict` is a parameter / field that can be used to influence the behavior of the `update` action.

**Review by danielmitterdorfer — CHANGES_REQUESTED** [review]
Looks mostly fine; there is just one place where we alter the meaning now. Can you please have a look?

File: docs/reference/docs/bulk.asciidoc [hunk 2]
```diff
@@ -246,20 +246,20 @@ NOTE: Only the shards that receive the bulk request will be affected by
...
-refresh. The other two shards of that make up the index do not
+refresh. The other two shards that make up the index do not
 participate in the `_bulk` request at all.
 
 [float]
 [[bulk-update]]
 === Update
 
-When using `update` action `retry_on_conflict` can be used as field in
+When using the `update` action, `retry_on_conflict` can be used as a field in
```
Comment (dmeiss) [inline docs/reference/docs/bulk.asciidoc]: I think this fixes it -- does it look OK to you?

File: docs/reference/docs/bulk.asciidoc (hunk 1 above)
Comment (dmeiss) [reply docs/reference/docs/bulk.asciidoc]: Fixed

File: docs/reference/docs/bulk.asciidoc (hunk 2 above)
Comment (danielmitterdorfer) [reply docs/reference/docs/bulk.asciidoc]: Looks fine to me, yes. Thank you.

**Review by danielmitterdorfer — APPROVED** [review]
Looks good to me.

**Comment by danielmitterdorfer** [comment]
@elasticmachine test this please

**Pull request merged**